tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot

```diff
@@ -145,7 +149,58 @@ Created Network And Images Persists As Well As Containers Are Discovered With Co
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${alpine}
     Should Be Equal As Integers  ${rc}  0
 
+    Cleanup VIC Appliance On Test Server
+
+Container on Open Network And Port Forwarding Persist After Reboot
+    Set Test Environment Variables
```

Comment:
Use `[Setup]  NONE` to suppress the normal appliance install instead of changing all the tests. Longer term we should provide Keywords for deploying VCHs of various standard configurations for easier implementation of these more involved setups.